autoware_adapi_v1_msgs/manual/msg/PedalsCommand.msg

```diff
@@ -1,3 +1,3 @@
 builtin_interfaces/Time stamp
-float32 accelerator
+float32 throttle
```

Comment:
How about `accel_throttle` and `brake_throttle`?

Reply:
The throttle seems to come from the engine's fuel throttle valve and appears to be used to mean accelerator. So I don't think this is suitable for brake.

There are other ways to call it, such as accelerator pedal or gas pedal, and considering EV, accelerator seems to be the most appropriate, but since it is confusing with acceleration, I have adopted throttle.

Reply:
If you want to avoid confusion with acceleration, maybe: 
* `throttle` and `brake`
* `throttle_pedal` and `brake_pedal`
might be also work.